server/integration_tests/test_data/e2e_edge_cases2/povertyvs.unemploymentrateindistrictsoftamilnadu/chart_config.json

```diff
@@ -31,12 +31,44 @@
             ],
             "denom": "Count_Person",
             "title": "Population Below Poverty Line"
+          },
```

Comment:
@pradh is this change valid?

Reply:
This part is valid and comes from new data addition.